### DISCUSSION TITLE
Company logo update in "Who Uses Renovate?" section

### DISCUSSION THREAD

**Discussion by siwinski** [renovatebot/renovate#15737] [Request Help] [answered]
Who can we (Red Hat) talk to regarding updating the Red Hat logo used in the "Who Uses Renovate?" section image [[readme](https://github.com/renovatebot/renovate/blob/main/readme.md?plain=1#L44)][[image](https://app.renovatebot.com/images/matrix.png)]?  Red Hat has a new logo that we would like to have used (see https://www.redhat.com/en/about/brand/standards/logo) instead of the one that is currently used.

**Comment by HonkingGoose** [comment, marked as answer]
**Edit:** We now use the new Red Hat logo.

---

@rarkins do you want to update the Red Hat logo in the `matrix.png` source image?

You may want to update all logos and be done with it. See issue:

- https://github.com/renovatebot/renovate/issues/10300

**Reply by rarkins** [reply]
Thanks, I'll take care of it

**Reply by rarkins** [reply]
Done

**Reply by siwinski** [reply]
Thanks!
